lib/actions/model.js

```diff
@@ -108,4 +108,11 @@ module.exports = {
       return validationErrors;
     };
   },
+
+  set(model, ...args) {
+    return () => {
+      model.set(...args);
```

Comment:
Can you be a bit more specific of what you'd like to see? As far as this code is concerned, the `model` here is just an object with a `set` function on it. We can change the name to something more general, but we thought since we were in `modelActions` this was fine.

At the end of the day, this function needs either an object with some sort of `set` function it can call, or it needs the function itself which knows how to set attributes on the model. We weren't sure which was better so we tried the former, but we can do the latter if that's preferable.

This change was requested by @naganowl as a generalization of this functionality: https://github.com/mavenlink/mavenlink/pull/8618/files/6af2b31968d15b17e1e21b4f57ec21c9413dcaa0..329e718e37ee4d9dc914ba15ed02ef274930bf0e#diff-0ead9fe0c37cd3dd88fbb1172467a673R44